src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -807,6 +807,43 @@ private void AddPythonEnginesOptions()
         /// </summary>
         public TrustedPathViewModel TrustedPathsViewModel { get; set; }
 
+        /// <summary>
+        /// Import Settings
+        /// </summary>
+        /// <param name="filePath"></param>
+        public void importSettings(string filePath)
+        {
+            PreferenceSettings newPreferences = dynamoViewModel.ImportPreferences(filePath);
+            newPreferences.CopyProperties(preferenceSettings);
+
+            // Explicit copy
+            preferenceSettings.SetTrustWarningsDisabled(newPreferences.DisableTrustWarnings);
+            preferenceSettings.SetTrustedLocations(newPreferences.TrustedLocations);
+            TrustedPathsViewModel?.InitializeTrustedLocations();
+
+            // Set the not explicit Binding
+            runSettingsIsChecked = preferenceSettings.DefaultRunType;
+            var engine = PythonEnginesList.FirstOrDefault(x => x.Equals(preferenceSettings.DefaultPythonEngine));
+            SelectedPythonEngine = string.IsNullOrEmpty(engine) ? Res.DefaultPythonEngineNone : preferenceSettings.DefaultPythonEngine;
+            dynamoViewModel.RenderPackageFactoryViewModel.MaxTessellationDivisions = preferenceSettings.RenderPrecision;
+            dynamoViewModel.RenderPackageFactoryViewModel.ShowEdges = preferenceSettings.ShowEdges;
+            PackagePathsForInstall = null;
+            PackagePathsViewModel?.InitializeRootLocations();
+
+            dynamoViewModel.IsShowingConnectors = preferenceSettings.ShowConnector;
+            dynamoViewModel.IsShowingConnectorTooltip = preferenceSettings.ShowConnectorToolTip;
+            foreach (Wpf.ViewModels.Watch3D.DefaultWatch3DViewModel item in dynamoViewModel.Watch3DViewModels)
+            {
+                Interfaces.BackgroundPreviewActiveState preferenceItem = preferenceSettings.BackgroundPreviews.Where(i => i.Name == item.PreferenceWatchName).FirstOrDefault();
+                if (preferenceItem != null)
+                {
+                    item.Active = preferenceItem.IsActive;
+                }                
+            }
+
+            RaisePropertyChanged("");
```

Comment:
This means?

Reply:
I could not find a lot of documentation on this behavior, but there are a few mentions here and there (fyi We do not support this behavior for our inhouse model notifications) 
Maybe we should use String.Empty  instead of ""

Reply:
Done.